opm/simulators/flow/FlowBaseVanguard.hpp

```diff
@@ -120,6 +125,16 @@ struct ZoltanParams {
     using type = UndefinedProperty;
 };
 
+template<class TypeTag, class MyTypeTag>
+struct MetisImbalanceTol {
+    using type = UndefinedProperty;
+};
+
```

Comment:
Do we really need a different one? Maybe we can have on to rule them all and deprecate the zoltan one?